docs/imagecustomizer/configuration.md

```diff
@@ -1746,5 +1766,101 @@ storage:
   resetPartitionsUuidsType: reset-all
 
 os:
-  resetBootLoaderType: hard-reset
+  bootloader:
+    resetType: hard-reset
+```
+
+### bootloader [[bootloader](#bootloader-type)]
```

Comment:
`bootloader` and `uki` are fields of `os type` not `storage type`.

Reply:
Same here, hopefully I put the right place this time.